docs/src/gcode/g-code.adoc

```diff
@@ -1766,7 +1766,7 @@ G73 X- Y- Z- R- Q- <L->
 
 The 'G73' cycle is drilling or milling with chip breaking.
 This cycle takes a Q number which represents a 'delta' increment along
-the Z axis.
```

Comment:
I suspect the "Rapid up .010 of an inch or 0.254 mm." part below should be modified too.  The documented  0.1"/0.254mm do not seem to match the ini value default listed above.